webdriver/tests/bidi/script/call_function/remote_values.py

```diff
@@ -39,3 +39,96 @@ async def test_remote_value_promise(bidi_session, top_context, await_promise):
         assert result == {"type": "number", "value": 42}
     else:
         assert result == {"type": "promise"}
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("await_promise", [True, False])
+async def test_window_top_level_browsing_context(bidi_session, top_context,
+                                                 await_promise):
+    function_declaration = f"()=>window"
```

Comment:
nit which applies to other tests as well:

```suggestion
    function_declaration = f"() => window"
```

Otherwise maybe return both the window and the frame's window within an array so that we have a single test only and can also check that both context values are different.